BetterPartyFinder/HookManager.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using Dalamud.Utility.Signatures;
+using FFXIVClientStructs.FFXIV.Client.UI.Agent;
+
+namespace BetterPartyFinder;
+
+// Code taken from: https://git.anna.lgbt/anna/XivCommon/src/branch/main/XivCommon/Functions/PartyFinder.cs
+public unsafe class HookManager
+{
+    [Signature("48 89 5C 24 ?? 48 89 74 24 ?? 57 48 83 EC 40 0F 10 81")]
```

Comment:
...github ate my comment...

is this signature likely to make its way into CS?